azure/provider_id_test.go

```diff
@@ -0,0 +1,23 @@
+package azure
+
+import (
+	"testing"
+)
+
+func TestParseProviderID(t *testing.T) {
+	var parseProviderIDTest = []struct {
+		providerID string
+		expected   string
+	}{
+		{
+			"",
+			"",
+		},
+	}
+
+	for _, tt := range parseProviderIDTest {
+		// do stuff
```

Comment:
did you mean to add this test in this PR?

Reply:
That was an accident, I didn't realize I'd merged with the branch this was on.